repo/ecc/ecc_rs_crc_test.go

```diff
@@ -0,0 +1,252 @@
+package ecc
+
+import (
+	"testing"
+
+	"github.com/kopia/kopia/repo/encryption"
+	"github.com/stretchr/testify/require"
+)
+
+/*
```

Comment:
should we uncomment this?

Reply:
The two commented methods are not really tests, they are just the code that I used to generate some of the tables above. I was not sure if it would be better to leave them there or just remove, but un-commenting does not really help. I think I'll just remove them.